README.md

```diff
@@ -1,47 +1,68 @@
-# Category Menu
+VTEX Category Menu
+=====
 
-Category Menu is a canonical app that any VTEX store can install.
+## Description
+The VTEX Category Menu app shows a department list of the store on an customizable menu. 
+This is a VTEX app that is used by Dreamstore product.
+
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
 
 ## Release schedule
 | Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
 | :--:     | :---:               |  :---:          | :---:                 | :---:       | :---: 
 | [1.x]    | **Maintenance LTS** |  2018-08-24     | 2018-11-20            | March 2019  | 1.x
 | [2.x]    | **Current Release** |  2018-11-20     |                       |             | 2.x
+See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for more information.
 
-
-## Continuous Integrations 
-
-### Travis CI 
-[![Build Status](https://travis-ci.org/vtex-apps/category-menu.svg?branch=master)](https://travis-ci.org/vtex-apps/category-menu)
+## Table of Contents
+- [Usage](#usage)
+- [API](#api)
+  - [Blocks API](#blocks-api)
+    - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+- [Troubleshooting](#troubleshooting)
 
 ## Usage
 
-TODO
+This app use our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
 
-### Running Tests
+To use this app you need to import in your dependencies on `manifest.json`.
 
-To run the test suit, type the following in the terminal, inside the folder `react`
-
-```sh
-$ npm t     # this
-$ yarn test # or this one
+```json
+  dependencies: {
+    "vtex.category-menu": "2.x"
+  }
 ```
 
-To update the tests snapshots use
+Then, add `category-menu` block into your app theme, like we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json). 
 
-```sh
-$ npm t -- -u  # like this
-$ yarn test -u # or this
-```
+### Blocks API
+:construction: :construction: :construction:
 
-## Configuration
+This app have an interface that describes what are the rules that the blocks must implement when you want to use the category-menu block. 
 
-TODO
+```json
+  "category-menu": {
+    "component": "index"
+  }
+}
+```
 
-## Troubleshooting
+#### Configuration 
+Through the Storefront you can change the behavior and interface of minicart. But, you also can make adjusts in your theme app, like Dreamstore does.
```

Comment:
Through the Storefront you can change the behavior and interface of minicart. But, you can also make adjusts in your theme app, like Dreamstore does.